### ISSUE TITLE
Update amber-website to 0.14.3

### ISSUE THREAD

**Issue by hhzl** [amber-smalltalk/amber-website#24]
Planned activity
1. fork repository https://github.com/amber-smalltalk/amber-website

```
git clone https://github.com/hhzl/amber-website
cd amber-website
git checkout -b gh-24
npm update amber-dev
bower install
git add -A
git commit -m "update to 0.14.3"
git push origin gh-24
```

Is this all that is needed?

**Comment by hhzl** [comment]
What I did

```
git clone https://github.com/hhzl/amber-website
cd amber-website/
bower install amber
git status
git add -A
git commit -m "updated to 0.14.3"
git push origin gh-pages
history
```

Result is http://hhzl.github.io/amber-website

The log of the command

```
    bower install amber
```

```
hhzl@hhzl-Latitude-E6410 ~/Documents/amber-website $ git branch
* gh-pages
hhzl@hhzl-Latitude-E6410 ~/Documents/amber-website $ bower install amber
bower amber#*                   cached git://github.com/amber-smalltalk/amber.git#0.14.3
bower amber#*                 validate 0.14.3 against git://github.com/amber-smalltalk/amber.git#*
bower amber#>=0.13.2            cached git://github.com/amber-smalltalk/amber.git#0.14.3
bower amber#>=0.13.2          validate 0.14.3 against git://github.com/amber-smalltalk/amber.git#>=0.13.2

Unable to find a suitable version for amber, please choose one:
    1) amber#^0.13.3 which resolved to 0.13.3 and is required by trysmalltalk#0.0.2, trysmalltalkdemo 
    2) amber#>=0.13.2 which resolved to 0.14.3 and is required by helios#0.3.1 
    3) amber#~0.14.3 which resolved to 0.14.3Prefix the choice with ! to persist it to bower.json

? Answer:: !3
```

**Comment by hhzl** [comment]
This issue requires that https://github.com/amber-smalltalk/trysmalltalk/issues/17 has been done before.

**Comment by ghost** [comment]
@hhzl Not true. You should close #17, as it is not needed (trysmalltalk is perfectly ok as is, and uses 0.14.3 (yes, it uses 0.14.3, because it uses anything 0.14.x, at least 0.14.1, so it resolves to 0.14.3)). Problem is, you have wrong version of `trysmalltalk` here, in `amber-website` repo, in `bower_components`. That version is old and requires `^0.13.3`. Just update it with the actual one. You have version 0.0.2 here.

`bower install "trysmalltalk#*" --save` should do the trick (meaning: "get the latest and save the right incantation into `bower.json`).

They you can update amber and no confllct appears.

**Comment by tomrake** [comment]
My ignorance caused most of this confusion, I did not understanding the Carat ^ functions in specification of versions. Herby is right, trysmalltalk can be used at the current master at c3f493f3. https://github.com/amber-smalltalk/trysmalltalk/issues/17 can be abandoned

I will very soon offer a fast foward merge commit that should close this issue as well. See it at https://tomrake.github.io/learn.html for the trysmalltalk part of the website.
